src/pages/contributors/mark-waite.adoc

```diff
@@ -0,0 +1,85 @@
+Extreme Programming requires continuous integration.
+We set our goal to have "XP by 3/03" and started to create our own CI server.
+
+In 2007 or 2008, I switched to a new company with a new project and discovered the continuous integration server that became Jenkins.
+I was impressed at how easy it was to deploy and manage and how well it worked.
+
+A few companies after that, I joined CloudBees to work full-time on helping Jenkins grow.
+
+== How long have you been using Jenkins?
+
+I started using Jenkins in 2007 or 2008.
+I've been using it since then, even during the times when I worked for employers that chose another continuous integration solution.
+
+== Why choose Jenkins over other projects?
+
+Jenkins was easy to configure and easy to manage, especially compared to its competitors at the time.
+I defined jobs, modified configurations, and viewed test results from a web browser.
+I often found that the flexibility of Jenkins was more important to me than the specific reasons that an employer had chosen another solution.
+I had problems to solve and Jenkins solved those problems.
+
+== What problems has Jenkins solved for you?
+
+Jenkins builds my code, runs my tests, and shows the results across a wide range of systems and technologies.
+
+== Is there an aspect of Jenkins that you're particularly passionate about?
+
+I have been involved in each area of Jenkins at some point, but I'm proud of the link:https://plugins.jenkins.io/git/[Git] and link:https://plugins.jenkins.io/git-client/[Git client] plugins that I created and maintain.
+I'm also involved in the platforms and documentation SIGs, where I host the later Jenkins documentation office hours.
+
+== What sort of contributions have felt the most successful or impactful?
+
+The link:https://www.jenkins.io/doc/developer/tutorial-improve/[Improve a plugin tutorial] and the link:https://www.jenkins.io/blog/2023/11/06/introducing-2-2-2-java-support-plan/[2+2+2 Java Support Plan] have both felt like impactful and successful contributions. 
+In each case, they were important to a wide group of people and they were important for an extended period of time.
+
+== Advice for new developers and new members of the open-source community
+
+Do not in believe self-imposed barriers to open source contribution.
+I mistakenly thought that a mere mortal, such as myself, could not possibly be of use to a successful and impressive project like Jenkins.
+I was wrong.
+Mere mortals are _exactly_ the type of people who do _amazing_ work in the Jenkins project.
+Find a topic that interests you, either professionally or personally.
+Find areas in the Jenkins project that are involved in that area of interest.
+Contribute your skills and your passion in that area.
+Help others do the same.
+
+I was interested in test automation, so I wrote automated tests for the Jenkins git plugin.
```

Comment:
```suggestion
I was interested in test automation, so I wrote automated tests for the Jenkins Git plugin.
```
We usually initial cap references to Git in our docs. Please change throughout.

Reply:
thanks very much @ammanders!